.evergreen/config.yml

```diff
@@ -1184,15 +1215,29 @@ axes:
 
 
 buildvariants:
-# Test all PHP versions with latest-stable MongoDB on Debian 11 and RHEL 7.0
+# Test all PHP versions with latest-stable MongoDB on Debian 11, Debian 10, RHEL 8.0, Ubuntu 20.04
 - matrix_name: "test-php-versions"
-  matrix_spec: { "os": ["rhel70", "debian11"], "mongodb-edge-versions": "latest-stable", "php-versions": "*" }
+  matrix_spec:
+    os:
+    - debian11
+    - debian10
+    - rhel90
+    # TODO: RHEL 8.x is not working yet
+    # - rhel82-arm64
+    # - rhel80
+    - rhel76
+    - ubuntu2204-arm64
+    - ubuntu2204
+    - ubuntu2004-arm64
+    - ubuntu2004
```

Comment:
Is it redundant to test PHP 8.1+ on both ubuntu2204 and ubuntu2004 (x64 and arm64)?

To make more efficient use of Evergreen, perhaps we can split this around the version boundary, similar to what we do with debian92 and debian11 in the `test-mongodb-versions` matrix.

Reply:
Since this involves different versions of OpenSSL I found it prudent to test the full range. Also, only testing the old versions would have us continuously update the list of excluded PHP versions on Ubuntu 20.04 as new versions are added. I'm not that concerned about the load on Evergreen on checks running on our branches, but I think it would make sense to limit the tasks we run when testing pull request. I don't think we need to test the full complement of PHP versions on all operating systems for that.

Potentially, we can re-address this when we refactor our build matrices in PHPC-2201.